pgbouncer/Makefile

```diff
@@ -0,0 +1,32 @@
+VM_NAME := pgbouncer-sampleapp
+
+.PHONY: run stop exporter fetch-prometheus-metrics render-config launch-vm clean defaultconfig
+
+run: launch-vm
+	@echo "VM $(VM_NAME) launched and configured."
+
+stop:
+	@multipass stop $(VM_NAME)
+	@multipass delete $(VM_NAME)
+	@multipass purge
+
+fetch-prometheus-metrics:
+	@multipass exec $(VM_NAME) -- curl http://localhost:9127/metrics > prometheus_metrics
```

Comment:
Got a `curl: (7) Failed to connect to localhost port 9127 after 3 ms: Connection refused` when trying to fetch prometheus metrics.